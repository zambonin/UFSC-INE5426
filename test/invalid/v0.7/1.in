bool fun f()
bool fun f()
