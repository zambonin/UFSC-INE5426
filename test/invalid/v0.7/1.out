[Line 3] semantic error: re-definition of function f
[Line 3] semantic error: function f is declared but never defined
